helm/csi-vxflexos/Chart.yaml

```diff
@@ -1,17 +1,16 @@
----
-name: csi-vxflexos
-version: 2.0.0
-appVersion: 2.0.0
 apiVersion: v2
+appVersion: "2.0.0"
+kubeVersion: ">= 1.19.0 <= 1.22.0"
```

Comment:
please change this to 
kubeVersion: ">= 1.19.0 <= 1.23.0"

Reply:
changed it